web-server/v0.4/components/Layout/Summary.js

```diff
@@ -16,22 +18,39 @@ class Summary extends React.Component {
     this.state = {
       summaryResult: [],
       iterations: [],
+      iterationSearch: [],
       columns: [],
-      loading: true
+      configData: {},
+      selectedConfig: [],
+      responseData: {},
+      selectedPort: "all",
+      ports: [],
+      loading: true,
+      searchText: '',
+      activeTab: 'iterations'
     }
   }
 
    componentDidMount() {
     this.setState({loading: true});
+    const { result, controller } = this.props;
     axios.get('http://es-perf44.perf.lab.eng.bos.redhat.com:9280/dsa.pbench.*/_search?source={ "query": { "match": { "run.name": "' + this.props.result + '" } }, "sort": "_index" }').then(res => {
```

Comment:
Can we find a way to remove this hard-coded `http://es-perf44.perf.lab.eng.bos.redhat.com` URL Elasticsearch prefix?